internals/secrethub/service_ls.go

```diff
@@ -67,3 +93,46 @@ func (cmd *ServiceLsCommand) Run() error {
 
 	return nil
 }
+
+type serviceTable interface {
+	header() []string
+	row(service *api.Service) []string
+}
+
+type baseServiceTable struct{}
+
+func (sw baseServiceTable) header() []string {
+	return []string{"ID", "DESCRIPTION"}
+}
+
+func (sw baseServiceTable) row(service *api.Service) []string {
+	return []string{service.ServiceID, service.Description}
+}
+
+type keyServiceTable struct {
+	baseServiceTable
+}
+
+func (sw keyServiceTable) header() []string {
+	return append(sw.baseServiceTable.header(), "TYPE")
+}
+
+func (sw keyServiceTable) row(service *api.Service) []string {
+	return append(sw.baseServiceTable.row(service), string(service.Credential.Type))
+}
+
+type awsServiceTable struct {
+	baseServiceTable
+}
+
+func (sw awsServiceTable) header() []string {
+	return append(sw.baseServiceTable.header(), "ROLE", "KMS KEY")
```

Comment:
Will this display the role twice? Once in the `DESCRIPTION` and once in the `ROLE` column?

Reply:
If the service is created with the CLI and no description is given, yes, as the description defaults to the role.

Reply:
Okay, so think about that for a second. 

You're a user and you are creating a service and providing the role ARN. Then you run `secrethub service aws ls` and you see that long ARN printed twice. What's your initial reaction?

More confused or more like yay 🎉 ?

How can we improve the developer's user experience here?

Reply:
#145